_translator-files/po/de/Running-a-Server.po

```diff
@@ -192,111 +200,115 @@ msgid ""
 "\t<figcaption>How Unregistered Servers work</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"\t<img src=\"{% include img/en-screenshots/diagram-unreg-server.inc %}\" loading=\"lazy\" alt=\"Diagram of connections between Clients within a Jamulus Unregistered Server\">\n"
+"\t<figcaption>Wie unregistrierte - private Server funktionieren</figcaption>\n"
+"</figure>\n"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:70
 msgid "If you are running an unregistered server behind a home internet connection, you might need to enable [port forwarding](#port-forwarding) as described below."
-msgstr ""
+msgstr "Wenn du einen unregistrierten/privaten Server über eine private Internetverbindung betreibst, musst du möglicherweise die [Portweiterleitung](#port-forwarding) wie unten beschrieben aktivieren."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:72
 msgid "### 3. Directory"
-msgstr ""
+msgstr "### 3. Verzeichnis Server"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:74
 msgid "If you want to run a number of Servers, possibly also behind a firewall or on a LAN, you may want to run your Server as a Directory. Examples include online events, music associations, sectional rehearsals or music lessons for schools."
-msgstr ""
+msgstr "Wenn du mehrere Server betreiben willst, möglicherweise auch hinter einer Firewall oder in einem LAN, kannst du deinen Server als Verzeichnis Server betreiben. Beispiele sind Online-Veranstaltungen, Musikvereine, Sektionsproben oder Musikunterricht für Schulen."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:76
 msgid "To run a Directory [read this guide](Custom-Directories)"
-msgstr ""
+msgstr "So betreibst du einen Verzeichnis Server aus [lese diese Anleitung] (Custom-Directories)"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:79
 msgid "## Installation and Configuration"
-msgstr ""
+msgstr "## Installation und Konfiguration"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:81
 msgid "Most people run Jamulus as a \"pure\" Server on **hardware without audio** (e.g. on a 3rd party/cloud host) running Linux. The following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution using systemd. To run a server on Windows or on the desktop with a graphical user interface, [see this section](#servers-on-the-desktop)."
-msgstr ""
+msgstr "Die meisten Leute betreiben Jamulus als \"reinen\" Server auf **Hardware ohne Audio** (z. B. auf einem Drittanbieter/Cloud-Host) unter Linux. Die folgenden Schritte setzen voraus, dass du mit der Kommandozeile und Debian/Ubuntu oder einer ähnlichen Distribution mit systemd vertraut bist. Um einen Server unter Windows oder auf dem Desktop mit einer grafischen Benutzeroberfläche zu betreiben, [siehe diesen Abschnitt](#servers-on-the-desktop)."
```

Comment:
#servers-on-the-desktop

Reply:
Right - otherwise the links will be broken